<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoundEditor</class>
 <widget class="QWidget" name="SoundEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>514</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <widget class="QFrame" name="frameSoundPreview">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frameProperties">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>261</width>
         <height>512</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>261</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QGroupBox" name="groupBoxPlayback">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>241</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Playback</string>
        </property>
        <widget class="QRadioButton" name="radioButtonFileStreaming">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>File Streaming</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButtonMemoryStreaming">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>221</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Memory Streaming</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelSampleRate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>62</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Sample Rate:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="labelBitRate">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>93</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Bit Rate:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QComboBox" name="comboBoxSampleRate">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>60</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>5512</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>11025</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22050</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32000</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>44100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>48000</string>
          </property>
         </item>
        </widget>
        <widget class="QComboBox" name="comboBoxBitRate">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>90</y>
           <width>91</width>
           <height>22</height>
          </rect>
         </property>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>16</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>24</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>32</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>40</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>48</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>56</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>64</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>80</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>96</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>112</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>128</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>144</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>160</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>192</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>224</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>256</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>320</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>512</string>
          </property>
         </item>
        </widget>
       </widget>
       <widget class="QPushButton" name="pushButtonLoadSound">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>111</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Sound</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/icons/folder_open.png</normaloff>:/icons/folder_open.png</iconset>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonEditJson">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>121</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Edit JSON</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/icons.qrc">
          <normaloff>:/icons/script.png</normaloff>:/icons/script.png</iconset>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditName">
        <property name="geometry">
         <rect>
          <x>52</x>
          <y>10</y>
          <width>199</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelName">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>13</y>
          <width>46</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBoxProperties">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>241</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Properties</string>
        </property>
        <widget class="QLabel" name="labelLength">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>221</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Length: 00:00:00</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelExtension">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>221</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Extension: </string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
